### PR TITLE
Add tests to track system preferences

### PR DESCRIPTION
**Pull request by ollema**
to be honest - I am not 100% sure what I am doing BUT it seems to work!

should I add comments for each block/paragraph of code or is it self-documenting and clear enough?

### CODE REVIEW

File: vite.config.ts [hunk 1]
```diff
@@ -11,10 +11,10 @@ export default defineConfig({
 		// in-source testing
 		includeSource: ['src/**/*.{js,ts,svelte}'],
 		// Add @testing-library/jest-dom matchers & mocks of SvelteKit modules
-		setupFiles: ['./scripts/setupTest.ts'],
+		setupFiles: ['src/tests/setup.ts'],
```
Comment (huntabyte) [inline vite.config.ts:14]: Does this not work if you add it to the existing setup file that exists?

**Review by huntabyte — COMMENTED** [review]
Wow! Glad you got this to work!

Can it be added to the existing setupTest.ts file in the `scripts` dir? That file contains the sensible defaults I add to all SK projects if we ever need to use any of the svelte internals

File: vite.config.ts (hunk 1 above)
Comment (ollema) [reply vite.config.ts:14]: oh wow, I don't know how I missed that this files existed already, sorry for that. will refactor!

Comment (ollema) [reply vite.config.ts]: fixed now, I though `./scripts/setupTest.ts` was some generic example path and not actually something that existed 🤦

File: scripts/setupTest.ts [hunk 2]
```diff
@@ -3,9 +3,9 @@
 import { vi } from 'vitest';
 import type { Navigation, Page } from '@sveltejs/kit';
 import { readable } from 'svelte/store';
-import * as environment from '$app/environment';
-import * as navigation from '$app/navigation';
-import * as stores from '$app/stores';
+import type * as environment from '$app/environment';
```
Comment (ollema) [inline scripts/setupTest.ts:6]: @huntabyte had to do this to get CI checks to be happy - do you know if this makes sense? have you ran into this before?

**Review by huntabyte — APPROVED** [review]

**Pull request merged**